source/Module/STIG/Functions.Checklist.ps1

```diff
@@ -240,18 +240,36 @@ function New-StigCheckList
     $writer = [System.Xml.XmlWriter]::Create($OutputPath.FullName, $xmlWriterSettings)
 
     $writer.WriteStartElement('CHECKLIST')
-
-    #region ASSET
-
     $writer.WriteStartElement("ASSET")
 
+    try
+    {
+        if ($hostName -eq $env:COMPUTERNAME)
+        {
+            [string]$localMac = (Get-NetAdapter | Where-Object {$_.Status -eq "Up"} | Select-Object MacAddress | Select-Object -First 1).MacAddress
```

Comment:
Style guidelines require the use of named parameters, in this case both Where-Object and Select-Object need their respective named parameters specified.  Furthermore, piping to multiple select-objects in this case can be avoided by either grouping or creating an additional variable and storing what you need for selection.

Reply:
Added Named Parameters